plugin/src/Reconciler/init.lua

```diff
@@ -64,8 +71,29 @@ function Reconciler:applyPatch(patch)
 		end
 	end
 
+	local patchTimestamp = DateTime.now():FormatLocalTime("LTS", "en-us")
+
 	local unappliedPatch = applyPatch(self.__instanceMap, patch)
 
+	if self.__fetchOnPatchFail then
+		-- TODO Is it worth doing this for additions that fail?
+		-- It seems unlikely that a valid Instance can't be made with `Instance.new`
+		-- but can be made using GetObjects
+		if PatchSet.hasUpdates(unappliedPatch) then
+			local idList = table.create(#unappliedPatch.updated)
+			for i, entry in unappliedPatch.updated do
+				idList[i] = entry.id
```

Comment:
Should also check here that the unapplied updates are for properties, otherwise we may try to fetch instances for updates (like metadata) that GetObjects won't help with